examples/jsm/loaders/SVGLoader.d.ts

```diff
@@ -12,6 +12,8 @@ export interface SVGResult {
 }
 
 export interface StrokeStyle {
+	fill: string;
+	fillOpacity: number;
```

Comment:
I'm not sure this is right. This interface describes the object which is returned by `getStrokeStyle()`. Hence, only the following properties should be relevant:

https://github.com/mrdoob/three.js/blob/624485b85c39d41857b630bc9374cca637d6a0ab/examples/js/loaders/SVGLoader.js#L1284-L1290

Reply:
@Mugen87 Hm, you're probably right. I put those there, because they're part of path.userData.style at line 198 in webgl_loader_svg.html:
https://github.com/mrdoob/three.js/blob/624485b85c39d41857b630bc9374cca637d6a0ab/examples/webgl_loader_svg.html#L198

And SVGLoader.pointsToStroke's second parameter is StrokeStyle type.

So, should I remove those, then?

Reply:
I would say yes. The rest of the PR looks good!